projects/plugins/jetpack/extensions/blocks/subscriptions/class-jetpack-subscription-site.php

```diff
@@ -144,23 +144,39 @@ function ( $hooked_block, $hooked_block_type, $relative_position, $anchor_block
 							'justifyContent' => 'stretch',
 						),
 					);
+
 					if ( ! empty( $anchor_block['attrs']['layout']['type'] ) ) {
 						$attrs['layout']['type'] = $anchor_block['attrs']['layout']['type'];
 					}
 
+					// translators: %s is the name of the site.
+					$discover_more_from_text = sprintf( __( 'Discover more from %s', 'jetpack' ), get_bloginfo( 'name' ) );
+					$subscribe_text          = __( 'Subscribe to get latest posts to your email.', 'jetpack' );
+					$inner_content_begin     = <<<HTML
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"0px","bottom":"0px","left":"0px","right":"0px"},"margin":{"top":"32px","bottom":"32px"}},"border":{"width":"0px","style":"none"}},"className":"has-border-color","layout":{"type":"default"}} -->
+<div class="wp-block-group has-border-color" style="border-style:none;border-width:0px;margin-top:32px;margin-bottom:32px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px">
+	<!-- wp:separator {"style":{"spacing":{"margin":{"top":"var:preset|spacing|40","bottom":"64px"}}},"className":"is-style-wide"} -->
+	<hr class="wp-block-separator has-alpha-channel-opacity is-style-wide" style="margin-top:var(--wp--preset--spacing--40);margin-bottom:64px"/>
+	<!-- /wp:separator -->
```

Comment:
Alternatively to separator I also explored using top/bottom border for group block, but I didn't get it to look very good, and couldn't pick the colour.

Reply:
This `<hr>` element is quite hard to pick in the Site Editor since it's exactly on the block group border.

Reply:
Noticed the same. Also the less blocks there are, the easier it's for folks to modify.

I think it's fine for now and we can tweak.

Reply:
Maybe we should use a border, similar to how we're doing it for the Paywall block?

I'll experiment with it a bit next week.

Reply:
Fixed it by adding `5px` of padding to the group block;